src/main/java/com/redislabs/redistimeseries/MultiRangeParams.java

```diff
@@ -7,6 +7,9 @@
 
 public class MultiRangeParams {
 
+  private long[] timestamps;
+  private double[] values;
```

Comment:
```suggestion
    private double minFilter;
    private double maxFilter;
```

Reply:
@gkorland With `double[] values`, we can do `if (values != null)`. With `double` we can't do such thing.